src/components/AssetDownload.tsx

```diff
@@ -102,34 +60,79 @@ const AssetDownload: React.FC<IProps> = ({
   const Svg = svg
 
   return (
-    <Container shouldHide={shouldHide}>
-      <h4>{title}</h4>
-      <div>
-        {children && <ImageContainer>{children}</ImageContainer>}
+    <Container
```

Comment:
Wondering why did you choose to use a `Container` in this case since the content width will always be 100%.

With `Box` you will not need to use `maxW` and `minW`.
```suggestion
    <Box
```

Reply:
Me to! lol. Yeah Box is the cleaner component.